### ISSUE TITLE
[Proporsal] Add desktop icon finally

### ISSUE THREAD

**Issue by tujhen** [Lurkki14/tuxclocker#64]
I think, If I do all clear, TuxClocker doesen't have desktop icon, but have tray logo and logo for app.

My proporsal is in adding .desktop file and a couple of resized tuxclocker-logo.svg that can be loaded in /usr/share/icons/hicolor/.

Can you add this? Or maybe I can take this task, but seems need some instructions, how integrate it into project correct.

**Comment by Lurkki14** [comment]
It should be pretty easy to add, you'd just call meson's [install_data](https://github.com/Lurkki14/tuxclocker/blob/master/src/tuxclocker-qt/meson.build) [here](https://github.com/Lurkki14/tuxclocker/blob/master/src/tuxclocker-qt/meson.build) to install all the needed files

 Might add it myself at some point but I have quite a few coding tasks at the moment.

**Comment by pallaswept** [comment]
I already did this in the openSUSE package but now some lawyers won't let us build it lol. Regardless, I have the .desktop file here if you want it.
```
[Desktop Entry]
Categories=Settings;HardwareSettings;
Encoding=UTF-8
Name=tuxclocker
GenericName=Qt overclocking tool
Type=Application
Exec=tuxclocker-qt
Icon=nvidia-settings]
```

I chose the nvidia-settings icon because it's already there if you have the nvidia driver, it's just the nvidia logo, and it's widely themed, so if you have an icon theme then tuxclocker will get themed too.
I imagine an AMD app's icon might be a better alternative, depending on which plugins are installed.
